caffe2/ideep/operators/conv_op.cc

```diff
@@ -40,15 +40,20 @@ class IDEEPConvOp final : public IDEEPConvPoolOpBase {
         (cached_weights_descriptor_ != filter.get_descriptor());
     if (weights_changed && !training_mode_) {
       cached_weights_descriptor_ = filter.get_descriptor();
-      filter_ = filter;
+      auto filter_in = filter;
+      filter_in.make_group(group_);
```

Comment:
Why do we need this line? Isn't the shape of filter_ determined by the next line?

Reply:
Never mind. Looks like it's changing the descriptor underlying, which means it's needed:
https://github.com/intel/ideep/blob/4bd9a6800bf7db068187619e0582d34dec9651dc/include/ideep/tensor.hpp#L835